frontend/src/components/Application/index.js

```diff
@@ -21,8 +21,8 @@ const Application = ({ text, applicationId }) => {
       <CommitteeName>Søknad</CommitteeName>
       <Wrapper>
```

Comment:
Do you need this wrapper anymore, when there is only one element in it?

Reply:
It just sets some padding and stuff, but since the ReadMore component is used lots of other places I kind of don't want to move it there and change components all over the site